Minecraft_Quick_Homes/Minecraft_Quick_Homes.ahk

```diff
@@ -345,15 +386,394 @@ Wait_Until_Minecraft_Registers_Slash()
     return
 }
 
-;Main Function
-CaseSwitch := 0
-IFSHIFT := 0
-Current_Home_warp=0
+/*
```

Comment:
This paragraph needs to be updated. It was created before as a starting point on how I would make it work. 
It would be more helpful for the end user, if it were either removed, or replaced with an explanation of how it works.

Reply:
Removed paragraft. Added a `begin (most of the logic) update 4.0`, to add a mark of where a lot of the additions were made.